canonical-web-and-design/certification.ubuntu.com
# Add cookiepolicy and global nav modules

## Done

Add cookiepolicy and global nav modules

## QA

1. download this branch
2. dotrun
3. look at http://0.0.0.0:8034/
4. see that the global nav is up to date and the cookie policy works

Fixes #113
